features/caseworker_claims_list.feature

```diff
@@ -40,7 +41,7 @@ Feature: Caseworker claims list
      Then I should only see <archive_number> claims
       And the claim count should show <archive_number>
 
-     Examples:
+     Examples: List of defendants and the number of live/archive claims they're involved in
```

Comment:
awesome stuff @aliuk2012: am happy to merge as is but a few points I wanted to note/query for future iterations:

1. I like your suggested dir structure but note that not all advocate features are under the external_uses/advocates folder, as yet
2. similar for case workers
3. there is one feature for advocates situatuated under case workers dir - separate comment above